test/pretty-printer/ppTest.ml

```diff
@@ -16,12 +16,14 @@ let generator =
            | 0 ->
                oneof
                  [
-                   return Expr.Unit;
+                   return @@ Mtt.Location.locate Expr.Unit;
```

Comment:
Using `Location.locate` here makes the code irregular. Can we do the wrapping as you did in Ast.ml?

Reply:
This part has not been resolved yet.